components/ZilPay/zilpay-base.ts

```diff
@@ -5,336 +5,336 @@ import { toast } from 'react-toastify'
 import { updateShowZilpay } from '../../src/store/modal'
 
 type Params = {
```

Comment:
fix prettier config so everyone has the same and we avoid this changes.

Reply:
any edit in the code logic @devwistkey ?

Reply:
Hi I don't have any changes for the prettier and zilpay-base @tralkan . What prettier version are you using? Mine v2.6.2